doc/Possible-key-values.md

```diff
@@ -32,24 +32,29 @@ Key values can be any of the following:
   + `CA:ctrl,a,ctrl,c` A key with legend CA that sends the sequence `ctrl+a`, `ctrl+c`.
   + `Cd:ctrl,backspace` A key with legend Cd that sends the shortcut `ctrl+backspace`.
 
-## Escape codes
-Value | Escape code for
+### Escape codes
+
+When defining a key value, several characters have special effects. If you want a character not to have its usual effect but to be taken literally, you should "escape" it by typing it in a special way:
+
+To get this character... | ...you can type
 :---- | :------
-`\?`  | `?`
-`\#`  | `#`
-`\@`  | `@`
-`\n`  | Literal newline character. This is different from `enter` and `action` in certain apps.
-`\t`  | Literal tab character. This is different from `tab` in certain apps.
-`\\`  | `\`
+`?` | `\?`
+`#` | `\#`
+`@` | `\@`
+A literal newline character, which is different from `enter` and `action` in certain apps. | `\n`
+A literal tab character, which is different from `tab` in certain apps. | `\t`
+`\` | `\\`
 
-XML escape codes also work, including:
+Certain other characters have special effects in the XML language. To have these characters taken literally, you can escape them in the usual way for XML:
```

Comment:
That's also the case for `\n`, `\t` and `\\` in a sense. What do you think of merging the two tables ?